test/presubmit-integration-tests-latest-release.sh

```diff
@@ -19,7 +19,7 @@
 # We currently take 0.10.0 for Serving and 0.10.2 for Eventing as the
 # latest release version.
 
-export KNATIVE_SERVING_VERSION="0.11.0"
+export KNATIVE_SERVING_VERSION="0.12.0"
```

Comment:
Does this belong here? Seems like #618 is more appropriate.

Reply:
True, but as we are going to merge #618 soon anyway, I don't mind much.

Reply:
I'd created this PR before 618 with version `0.11.1`. :-)
Let me take this off completely.